geomstats/geometry/hyperbolic.py

```diff
@@ -83,41 +95,48 @@ def __init__(self, dimension, point_type='extrinsic', scale=1):
 
     @staticmethod
     def _belongs_ball(point, tolerance=TOLERANCE):
-        """Evaluate if a point belongs to the Hyperbolic space (poin. ball).
+        """Test if a point belongs to the hyperbolic space.
 
-        Evaluate if a point belongs to the Hyperbolic space based on
+        Test if a point belongs to the hyperbolic space based on
         the poincare ball representation, i.e. evaluate if its
-        squared norm is lower than one.
+        squared norm is lower than 1.
 
         Parameters
         ----------
         point : array-like, shape=[n_samples, dimension]
-                Input points.
+            Point.
         tolerance : float, optional
+            Tolerance at which to evaluate how close is the squared norm
+            compared to 1.
 
         Returns
         -------
         belongs : array-like, shape=[n_samples, 1]
+            Array of booleans evaluating if the corresponding points
+            belong to the hyperbolic space.
         """
         return gs.sum(point**2, -1) < (1 + tolerance)
 
     def belongs(self, point, tolerance=TOLERANCE):
-        """Evaluate if a point belongs to the Hyperbolic space.
+        """Test if a point belongs to the hyperbolic space.
 
-        Evaluate if a point belongs to the Hyperbolic space according
-        to the current representation
+        Test if a point belongs to the hyperbolic space according
+        to the current representation.
 
         Parameters
         ----------
-        point : array-like, shape=[n_samples, dimension] or
-                shape=[n_samples, dimension + 1] for extrinsic
-                coordinates
-                Input points.
+        point : array-like, shape=[n_samples, dimension]
+                            or shape=[n_samples, dimension + 1]
+            Points.
         tolerance : float, optional
+            Tolerance at which to evaluate how close is the squared norm
```

Comment:
"how close the squared norm is to the reference value"?

Reply:
Yes, it is better! Thank you!